src/WebJobs.Script.Grpc/Abstractions/WorkerDescriptionExtensions.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs.Script.Abstractions
+{
+    public static class WorkerDescriptionExtensions
+    {
+        public static bool IsValid(this WorkerDescription workerDescription)
```

Comment:
nit: this never returns `false`, so we either need to change the implementation to return a validation result or change the name to make it clear that this will throw on error.

Reply:
Will update the name